### ISSUE TITLE
Needs license.

### ISSUE THREAD

**Issue by DavidEBest** [USDepartmentofLabor/Child-Labor#71]
There should be an explicit license file in the repo, even if it is in the public domain.

**Comment by aquilent-dgemoets** [comment]
DOL will have to choose a license.  The GitHub license picker provides some templates that can be committed to your repository:
1.     browse to your repository at GitHub.com
2.     create a new file by pressing the blue + icon (updated to: New File button)
3.     name it LICENSE.md or LICENSE.txt to show up the license picker again
4.     choose a template

**Comment by jfc3-dol** [comment]
This must have been done because the application has been deployed a few times since this issue was created.

**Comment by anselmbradford** [comment]
Nope, not done. The repo should have a `LICENSE.md` file in root directory of  https://github.com/USDepartmentofLabor/Child-Labor, which it does not have.

See https://help.github.com/articles/licensing-a-repository/

**Comment by jfc3-dol** [comment]
Okay, we can re-open the issue.

I assumed it had been done but not note/closed in GitHub.
